### DISCUSSION TITLE
How to embed video ?

### DISCUSSION THREAD

**Discussion by hackeranmol** [toeverything/AFFiNE#8672] [Q&A]
Local video embed feature not available in AFFiNE? 

This is a must have for me. Please any other alternative that supports local video embed on a infinite canvas ?

**Comment by HeJiachen-PM** [comment]
You simply insert a "file" block, and then select your video to upload from it and change it to "embed view"

**Reply by hackeranmol** [reply]
I upload the video file but there is no option to change into embed view 

The video is only there as attachment

**Comment by HeJiachen-PM** [comment]
Sorry, what version are you using?

Anmol ***@***.***>于2024年11月6日 周三15:44写道：
